frontend/src/components/CustomTable.tsx

```diff
@@ -224,8 +226,10 @@ export default class CustomTable extends React.Component<CustomTableProps, Custo
 
     this.state = {
       currentPage: 0,
-      filterString: '',
-      filterStringEncoded: '',
+      filterString: this.props.initialFilterString || '',
+      filterStringEncoded: this.props.initialFilterString
+        ? this._createAndEncodeFilter(this.props.initialFilterString)
+        : '',
```

Comment:
Add a test for `initialFilterString` and `setFilterString` use case?

Reply:
@zijianjoy Should I do this using the existing `enzyme` setup detailed at [`CustomTable.test.tsx`](https://github.com/zpChris/pipelines/blob/master/frontend/src/components/CustomTable.test.tsx), or add a new `react-testing-library` test to that file?

Reply:
If feasible, we should use `react-testing-library`. Feel free to update here if not, then we can consider using enzyme.

Reply:
Sounds good - I've updated the tests here: https://github.com/kubeflow/pipelines/pull/8118/commits/8560ecccd4d131b72a8433aa7cda899b03f8a545, https://github.com/kubeflow/pipelines/pull/8118/commits/2ff93297a616b2828f10ca3e03a0997209dc41a2.